content/topics/intro-to-css/_index.md

```diff
@@ -0,0 +1,9 @@
+---
+title: Introduction to CSS
+ready: true
+---
+
+## Readings
+
+1. [A developer's guide to web design for non-designers](https://www.freecodecamp.org/news/a-developers-guide-to-web-design-for-non-designers-1f64ce28c38d/)
+1. [7 principles of design for coders](https://learntocodewith.me/posts/7-principles-of-design/)
```

Comment:
These readings look like they belong in here `topics/intro-to-web-design` 

Also naming conventions. If we have a topic called "intro to the letters of the alphabet" then it should not just include links to the complete works of Shakespeare. It should probably have content that introduces the letters of the alphabet.

Similarly, if we have a topic called "intro to css" then it should probably include introductory css resources. 

Names are important to coders. Call it what it is.